src/Templates/src/templates/maui-mobile/.template.config/localize/templatestrings.es.json

```diff
@@ -3,7 +3,13 @@
   "name": "Aplicación .NET MAUI",
   "description": "Un proyecto para crear una aplicación .NET MAUI para iOS, Android, Mac Catalyst, Tizen y WinUI",
   "postActions/openInEditor/description": "Abre MainPage.xaml en el editor.",
+  "postActions/restore/description": "Restore NuGet packages required by this project.",
+  "postActions/restore/manualInstructions/default/text": "Run 'dotnet restore'",
   "symbols/applicationId/description": "Reemplaza $(ApplicationId) en el proyecto",
   "symbols/Framework/description": "The target framework for the project.",
-  "symbols/Framework/choices/DOTNET_TFM_VALUE/description": "Target DOTNET_TFM_VALUE"
+  "symbols/Framework/choices/DOTNET_TFM_VALUE/description": "Target DOTNET_TFM_VALUE",
+  "symbols/IncludeSampleContent/displayName": "_Include sample content",
+  "symbols/IncludeSampleContent/description": "Configures whether to add sample pages and functionality to demonstrate basic usage patterns.",
+  "symbols/Empty/description": "Configures whether to omit sample pages and styling that demonstrate basic usage patterns.",
```

Comment:
Will it be located later? only the main description seems to be localized.

Reply:
@rmarinho I think a follow-up pr comes to localize, correct?

Reply:
yap when is merged to main it will trigger this localized we can then backport if we get the translations in time